launch_testing/package.xml

```diff
@@ -3,9 +3,9 @@
 <package format="2">
   <name>launch_testing</name>
   <version>0.7.3</version>
-  <description>ROS launch testing suite.</description>
-  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <description>A package to create tests which involve launch files and multiple processes.</description>
   <maintainer email="pete.baughman@apex.ai">Pete Baughman</maintainer>
+  <author email="pete.baughman@apex.ai">Pete Baughman</author>
```

Comment:
Dirk should remain an author here. As a general rule, I would not ever remove a maintainer without placing them as an author (even in a case like this were much or all of the original code has been replaced).

Also, having pete as an author and maintainer is a bit redundant, usually we don't do that, but it's ok as-is, just thought I'd mention it for future cases.

Reply:
Alright, so I'll put Dirk and Esteve back as authors, and make Pete a maintainer only.